CHANGELOG.md

```diff
@@ -1,3 +1,65 @@
+# OP-TEE - version 1.1.0
+
+
+## New features
+
+* Softfloat library: floating point support is now available in 32bits TA.
+
+* Support running 64-bits TA: on ARMv8 platform, TA can be compiled in
+  AArch32 and/or in AArch64 in case the core is compiled in AArch64.
+  An example can be found in HiKey configuration file, within the excerpt code
+
+    ta-targets = ta_arm32
+    ta-targets += ta_arm64
+
+  In such a case, the user TA libraries are compiled in both AArch32 and
+  AArch64, and can be found in out/arm-plat-hikey/export-ta_arm32 and
+  out/arm-plat-hikey/export-ta_arm64
+
+* Concurrent TA support: multiple TA can run in parallel on
+  several cores.
+
+* New tests added in xtest test suite: concurrent TA (xtest 1013),
+  floating point tests (xtest 1006 and os_test TA) and corruption
+  file storage (xtest 20000)
+
+* [Link][github_commits_1_1_0] to a list of all commits between this and
```

Comment:
I like this, but there is no diff when checking the URL. I haven't checked if you have made the actual tag or not. I you haven't then that explains it and it should work when you've done so.